[test] Place test constants in travis.xml.dist and travis.xml.travis

Maks3w opened the pull request:
Close #7383

Maks3w committed:
[test] Remove output buffering legacy setting

Maks3w committed:
[test] Remove generate report

Maks3w committed:
[test] Place test constants in travis.xml.dist and travis.xml.travis

weierophinney commented on tests/phpunit.xml.dist:
```diff
@@ -32,6 +32,451 @@
+        <!--
+          Zend\Auth\Adapter\DbTable tests
+        -->
+        <cons name="TESTS_ZEND_AUTH_ADAPTER_DBTABLE_PDO_SQLITE_ENABLED" value="false"/>
```
These should be "const", not "cons".

weierophinney commented on tests/phpunit.xml.travis:
```diff
@@ -0,0 +1,510 @@
+        <!--
+          Zend\Auth\Adapter\DbTable tests
+        -->
+        <cons name="TESTS_ZEND_AUTH_ADAPTER_DBTABLE_PDO_SQLITE_ENABLED" value="true"/>
```
Same comment: `s/cons/const/` in all of these declarations.

Maks3w committed:
[test] Add XSD schemas to PHPUnit

Maks3w committed:
[test] Fix settings with null value

Maks3w committed:
[code][test] Provide realpaths instead rely on include path

previously the already removed test/Bootstrap.php had legacy include_path directive

Maks3w commented:
@weierophinney Ready for merge

Maks3w commented:
@weierophinney I've removed `TEST_ZEND_DB` constants.
